src/apps/common/ns_turn_utils.c

```diff
@@ -38,7 +38,11 @@
 
 #include <pthread.h>
 
-#include <syslog.h>
+#if defined(__unix__) || defined(unix) || defined(__APPLE__) \
```

Comment:
For macOS I would only keep `defined(__APPLE__)`
All the rest would be confusing (and good luck to anyone trying to build the code for non-macOS Apple platform)

Also not sure if `defined(unix)` is needed. If not let's omit it.

Reply:
I think to keep it. We aren't clear why the author use it. I believe the author did this for a reason.
__DARWIN__ and __MACH__ mybe used at cross-compile time.

Unless we redefine the operating system it runs.